internal/ddt/node_doc_transformer_test.go

```diff
@@ -152,6 +153,7 @@ func TestNodeLinksKeepFragmentAndQueryFromOriginal(t *testing.T) {
 	dt, _ := NewNodeDocTransformer("/tree", "foo/bar", get, "test")
 
 	expected := map[string]string{
+		"<a href=\"./?answer=42\"></a>":      		"<a href=\"/tree/foo/bar?answer=42&v=test\" data-node=\"foo/bar\"></a>",
```

Comment:
When using `href="?answer=42"` is also accepted, please add an additional test for that.

Reply:
Good Idea